src/store/care/sagas/care.sagas.ts

```diff
@@ -0,0 +1,20 @@
+import { takeEvery, put, call } from 'redux-saga/effects';
+import { getPets } from '../actions/care.actions';
+
+function fetchPets() {
+  return fetch('https://fathomless-ridge-53873.herokuapp.com/pets').then(response => response.json());
```

Comment:
The `https://fathomless-ridge-53873.herokuapp.com` should be moved to the shared constants. I would expect that you are going to use it like `${api}/pets`

Reply:
As you can see this is not the final version of the page, 
so I will make these changes (and other) in the next pull request.